README.md

```diff
@@ -91,7 +89,7 @@ function stringsStringStrings(p1, p2, p3, p4="test") {
 }
 ```
```

Comment:
I would also add an example of Flow's comment types: https://flow.org/en/docs/types/comments/

```
function f(value /*: MyAlias */) /*: boolean */ {
  return value.bar;
}
```

Reply:
> ## In readme:
> > Type Assertions
> 
> `as` is TypeScript specific. You can add an example from Flow for our casting syntax: https://flow.org/en/docs/types/casting/
> 
> ```
> (x: T);
> ```
> 
> > One can explicitly specify the type arguments of a generic function invocation or generic class instantiation [in TypeScript](https://www.typescriptlang.org/docs/handbook/2/functions.html#specifying-type-arguments).
> 
> You can do the same in Flow, you can make this language more generic. https://flow.org/en/docs/types/generics/
> 

The first two comments have already been addressed. I suggest opening an issue or pull request to address the remaining comments. Do you want me to help or open those?